src/components/StatsInnerList.tsx

```diff
@@ -53,11 +60,18 @@ export default class StatsInnerList extends React.Component<StatsInnerListProps,
     return <ul className="stats-inner-list">{list}</ul>;
   }
 
+  guessYear(library: LibraryData): string {
+    let year = library.basic_info.timestamp && library.basic_info.timestamp.match(/20\d+/)[0];
```

Comment:
Can this be shortened to `const year = library.basic_info?.timestamp.match(/20\d+/)[0];`?

Reply:
Sure!  (Putting the question mark after `timestamp`, though; there shouldn't be any way for `basic_info` to be missing.)